### DISCUSSION TITLE
View all descendants for chosen comment

### DISCUSSION THREAD

**Discussion by brainchild0** [Docile-Alligator/Infinity-For-Reddit#1441] [Ideas]
# Description

When a comment tree is shown, to reveal part of a discussion, the view may show only one particular child for some comment, even though others have been given.

Presently, it appears impossible to select some particular comment for a new view that includes all of its children, and descendants.

# Alternatives

Presently, browsing an entire post discussion, and manually seeking the target comment, appears the only means to achieve a similar outcome.

# Context

Querying other children of some particular visible comment is an objective naturally occuring through normal usage.

# Importance

*Nice to have*

# Solution

Suggested is a feature for opening a new view showing a full descendant tree for some comment that may be selected from a previous view.

**Comment by shreya-shah** [comment]
This can also help for comments which can be deeply nested
